react-bootstrap/react-bootstrap.d.ts

```diff
@@ -128,7 +128,7 @@ declare module "react-bootstrap" {
     interface MenuItemClass extends React.ComponentClass<MenuItemProps> { }
     var MenuItem: MenuItemClass;
 
-
+	type renderable = string | number | Element;
```

Comment:
Yes Paul, based on the description [here](https://facebook.github.io/react/docs/reusable-components.html) the explanation says that anything which is of type _node_ will be 
```
 // Anything that can be rendered: numbers, strings, elements or an array
 // (or fragment) containing these types.
 optionalNode: React.PropTypes.node,
```
which is the type I have tried to create